libs/sdk-ui-filters/src/RankingFilter/MeasureDropdown/MeasureDropdownBody.tsx

```diff
@@ -12,6 +12,7 @@ interface IMeasureDropdownBodyProps {
     onClose: () => void;
```

Comment:
Please update the copyright of this file as well. As we are making some changes now, the copyright should change accordingly, shouldn't it?

`// (C) 2020 GoodData Corporation`

↓

`// (C) 2020-2022 GoodData Corporation`

Reply:
fixed